PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -111,7 +110,7 @@ public final class DefaultControlsViewController: ContentControlsViewController
                           controlsViewVisible: controlsShouldBeVisible)
         
         controlsView.isHidden = uiProps.controlsViewHidden
-        if #available(iOS 11.0, *)  {
+        if #available(iOS 11.0, *) {
             if controlsView.isHidden {
                 shouldHideHomeIndicator = true
                 setNeedsUpdateOfHomeIndicatorAutoHidden()
```

Comment:
Code duplication

Reply:
@invalid404 probably a bug after rebase, because this PR is frozen.